docs/README.md

```diff
@@ -30,13 +30,18 @@ The following CSI gRPC calls are **not yet** implemented:
-| "encrypted"                 |                       |         | Not supported                                                                 |
-| "kmsKeyId"                  |                       |         | Not supported                                                                 |
+| Parameters                                       | Values                | Default | Description                                                                                                                                      |
+| ------------------------------------------------ | --------------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------ |
+| "csi.storage.k8s.io/fstype"                      | xfs, ext2, ext3, ext4 | ext4    | File system type that will be formatted during volume creation                                                                                   |
+| "type"                                           | io1, gp2, standard    | gp2     | BSU volume type                                                                                                                                  |
+| "iopsPerGB"                                      |                       |         | I/O operations per second per GiB. Required when io1 volume type is specified                                                                    |
+| "encrypted"                                      | "true", "false"       | "false" | Specify if we want to encrypt te disk or not                                                                                                     |
+| "csi.storage.k8s.io/node-stage-secret-name"      | string                |         | The name of the secret  (See [template](https://kubernetes-csi.github.io/docs/secrets-and-credentials-storage-class.html#node-stage-secret))     |
+| "csi.storage.k8s.io/node-stage-secret-namespace" | string                |         | The namespace of the secret (See [template](https://kubernetes-csi.github.io/docs/secrets-and-credentials-storage-class.html#node-stage-secret)) |
+| "kmsKeyId"                                       | string                |         | Not yet supported                                                                                                                                |
+| "luks-cipher"                                    | string                |         | LUKS encryption cipher to use  (See [doc](https://wiki.archlinux.org/title/dm-crypt/Device_encryption#Encryption_options_for_LUKS_mode))                    |
+| "luks-hash"                                      | string                |         | Derivation Password hash algorithm (See [doc](https://wiki.archlinux.org/title/dm-crypt/Device_encryption#Encryption_options_for_LUKS_mode))     |
```

Comment:
Maybe mention that the default hash and cypher depends of installed luks version.
Also, maybe point to official documentation if possible (https://gitlab.com/cryptsetup/cryptsetup/blob/master/README.md)